src/client.ts

```diff
@@ -45,6 +48,11 @@ export interface TequilapiClient {
   connectionCancel (): Promise<void>,
   connectionIP (timeout?: number): Promise<ConnectionIPDTO>,
   connectionStatistics (): Promise<ConnectionStatisticsDTO>,
-  location (timeout?: number): Promise<ConsumerLocationDTO>,
-  sessionsList (): Promise<SessionDTO[]>
+
+  sessionsList (): Promise<SessionDTO[]>,
+
+  serviceList (): Promise<ServiceInfoDTO[]>,
+  serviceGet (serviceId: string): Promise<ServiceInfoDTO>,
+  serviceStart (request: ServiceRequest, timeout?: number): Promise<ServiceInfoDTO>,
+  serviceStop (serviceId: string): Promise<void>
```

Comment:
To make it more consistent with other endpoints:
```suggestion
  serviceStop (id: string): Promise<void>
```

Reply:
Same here